### PR TITLE
Register infinotiver.is-a.dev and delete prakarsh.is-a.dev

### PR DESCRIPTION
**Pull request by infinotiver**
## Requirements

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview
image

## Delete prakarsh.is-a.dev
Pls refer to https://github.com/is-a-dev/register/issues/11369

### CODE REVIEW

File: domains/infinotiver.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "prakarsh17",
+        "email": "prakarsh.3482@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://prakarsh17.github.io/profile/"
```
Comment (Stef-00012) [inline domains/infinotiver.json:7]: Invalid CNAME record (can not have `https://` or `/`)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by infinotiver — COMMENTED** [review]
Updated website record (removed // and /)

**Comment by prakarsh17** [comment]
Fixed and merged "main" into main (updated branch)..
All issues must be solved now??!

**Review by infinotiver — COMMENTED** [review]
Updated my username - changed info in infinotiver.json

**Review by Stef-DP — APPROVED** [review]
seems good now

**Review by iostpa — APPROVED** [review]
seems fine to me

**Pull request merged**